spec/index.html

```diff
@@ -1022,34 +1022,19 @@ <h2>Embedding YAML-LD in HTML Documents</h2>
      title="YAML-LD document embedded in HTML">
     <!--
     <script type="application/ld+yaml">
-    "@context": https://json-ld.org/contexts/person.jsonld
-    "@id": http://dbpedia.org/resource/John_Lennon
-    name: John Lennon
-    born: 1940-10-09
-    spouse: http://dbpedia.org/resource/Cynthia_Lennon
-    ---
-    "@context": https://json-ld.org/contexts/person.jsonld
-    "@id": http://dbpedia.org/resource/Cynthia_Lennon
-    born: 1939-09-10
+      "@context": https://json-ld.org/contexts/person.jsonld
+      "@id": http://dbpedia.org/resource/John_Lennon
+      name: John Lennon
+      born: 1940-10-09
+      spouse: http://dbpedia.org/resource/Cynthia_Lennon
     </script>
     -->
     </pre>
 
     <p data-tests="manifest.html#html-dedent-needed,manifest.html#html-dedent-not-needed">
-      YAML syntax is indentation based. Before processing each `&lt;script>` block with YAML-LD content, YAML-LD processor MUST <em>dedent</em> the body of the block using the following procedure:
+      YAML syntax is indentation based. When processing each `&lt;script>` block with YAML-LD content, YAML-LD processor MUST <em>preserve</em> all space and newline characters in the block content as they are.
```

Comment:
Is it only spaces and newlines that must be preserved? What about tabs (other than as indentation), non-breaking spaces, control characters, and other whitespace? (I tried to RTFM, but it didn't say anything except that "[YAML is whitespace-sensitive and indentation defines the structure, but it doesn’t accept tabs for indentation.](https://docs.platform.sh/learn/overview/yaml/what-is-yaml.html)")

Reply:
What about this? We now say that the processor shouldn't modify the content of the `<script>` block. Which doesn't actually bring new information, I guess. Instead of saying "trim and decent" we're saying "do not trim, do not dedent".